.gitignore

```diff
@@ -44,3 +44,15 @@ contrib/local-test/query.gql
 contrib/local-test/schema.dql
 contrib/local-test/schema.graphql
 contrib/local-test/variables.json
+
+# Test logs and test files
+*.enc
+
+systest/bulk_live/live/data/*
+systest/bulk_live/bulk/data/*
+
+*/t
```

Comment:
@matthewmcneely We should rename this package. Why "t"? is that a weird convection?

This PR looks like it will take some time, we can wait for a decision.

Reply:
That folder's been named `t` as long as I've been familiar with the codebase. Not exactly a descriptive package name, no. Probably easier to change the const declaration of Dgraph's temporary folder name, say maybe to `tmp`? Would have to investigate impacts of this of course.

Reply:
Is this referring to the `t` directory in our repo containing the `t.go` test script? If so, I think we can just change it to a more descriptive name like `test` or something, there shouldn't be a lot of changes needed for this.

Reply:
@kevinmingtarja that would be nice. To be human readable.

I could also remove this ignore from root and let it just for /dgraph/dgraph/* cuz it is there that we expect to see the "t" folder. Which is a tmp folder from Dgraph binary.

Also, why naming has to be so short? everywhere? is that some minimalist convention? odd.